webapp/utils.py

```diff
@@ -94,3 +94,19 @@ def _validate_num(num, default):
     pages = int(math.ceil(float(len(input_list))/page_size))
     result_list = input_list[start:end]
     return (result_list, {"page": page, "page_size": len(result_list), "pages": pages})
+
+
+def filter_item_if_exists(list_to_process, item_details):
+    """
+    Filter to check if item exists
+    :param item_details: item details from cache
+    :param list_to_process: list of items to filter
+    :return: filtered list of items
+    """
+
+    filtered_list_to_process = []
+    for item in list_to_process:
+        item_detail = item_details.get(item)
+        if item_detail:
```

Comment:
In this case I would write just:
~~~python
if item in item_details:
    filtered_list_to_process.append(item)
~~~

Or it could be all written in one line just using collection transformation:
~~~python
[item for item in list_to_process if item in item_details]
~~~